_sass/_mosaic.scss

```diff
@@ -0,0 +1,55 @@
+.mosaic-container {
+  --_mosaic--grid-size: 8;
+  --_mosaic--gap: clamp(4px, 2cqi, 16px);
+  --_mosaic--cell-size: calc(100cqi / var(--_mosaic--grid-size));
+
+  display: grid;
+  grid-template-columns: repeat(8, var(--_mosaic--cell-size));
+  grid-template-rows: repeat(8, var(--_mosaic--cell-size));
+  place-content: center;
+  max-width: 800px;
+
+  margin: 0 auto;
+  
+  filter: drop-shadow(0 0 20px color-mix(in oklab, var(--show--color) 60%, transparent));
+}
+
+.mosaic-item {
+  --_mosaic-item--color: var(--mosaic-item--color, auto);
+  --_mosaic-item--position: var(--mosaic-item--position, auto);
+
+  grid-area: var(--_mosaic-item--position);
+
+  container-name: mosaic-item;
+  container-type: inline-size;
+
+  overflow: hidden;
+
+  margin: calc(var(--_mosaic--gap) / 2);
+
+  background-color: var(--_mosaic-item--color);
+
+  border-radius: calc(var(--_mosaic--cell-size) / 4);
+
+  &[data-color="blue"] {
+    --mosaic-item--color: #{$blue};
+  }
+  &[data-color="navy"] {
+    --mosaic-item--color: #{$navy};
+  }
+  &[data-color="cyan"] {
+    --mosaic-item--color: #{$cyan};
+  }
+  &[data-color="orange"] {
+    --mosaic-item--color: #{$orange};
+  }
+  &[data-color="yellow"] {
+    --mosaic-item--color: #{$yellow};
+  }
+}
+
+.mosaic-item img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
```

Comment:
OUTSTANDING! Really 🏆 

One request to ensure the show logo is always square. Some logos might not be as amenable as TW to the horizontal crop.